omf-python/src/geometry.rs

```diff
@@ -59,16 +59,23 @@ pub struct PySurface(pub Surface);
 #[pymethods]
 impl PySurface {
     #[getter]
+    /// Origin of the surface relative to the project coordinate reference system.
```

Comment:
Just thought I should mention somewhere, a caveat of the current way the python docstrings ares generated is that `pyo-stub-gen` doesn't translate getters to anything at the moment. I think because as it turns them into a variable and variables in the generated `omf_python.pyi` and in python variables don't have docstrings (at least, I think I tried adding them manually and didn't work).

If they changed it to instead generate python `@property` decorated methods I think they would have a place to go.

We could maintain `omf_python.pyi` manually but then its more likely to get out of sync with the code easily.